infoschema/cache_test.go

```diff
@@ -33,63 +33,77 @@ func (s *testInfoCacheSuite) TestInsert(c *C) {
 	c.Assert(ic, NotNil)
 
 	is2 := infoschema.MockInfoSchemaWithSchemaVer(nil, 2)
-	ic.Insert(is2)
-	c.Assert(ic.GetByVersion(2), NotNil)
+	ic.Insert(is2, 2)
+	c.Assert(ic.GetByVersion(2), DeepEquals, is2)
```

Comment:
If you want to assert equality, `SchemaMetaVersion(), Equals, xx.SchemaMetaVersion()` is enough.

Reply:
I think `DeepEquals` is more simple.